routes/web.php

```diff
@@ -93,8 +93,15 @@
 
     Route::get('designer/scripts/categories', [ScriptController::class, 'index'])->name('script-categories.index')->middleware('can:view-script-categories');
     Route::get('designer', [DesignerController::class, 'index'])->name('designer.index');
-
-    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'index'])->name('processes.catalogue.index')->middleware('can:view-process-catalog');
+    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
+        ->name('processes.catalogue.index')
+        ->middleware('can:view-process-catalog');
+    Route::get('process-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
```

Comment:
remove this Route::get('process-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
        ->name('process.catalogue.index')
        ->middleware('can:view-process-catalog');